### DISCUSSION TITLE
Sonoma-OC-1.0.1-Beta1_Release-Infinix-InBook-X1 Slim

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#39] [Discuss about the release]
### This is the first release of Infinix InBook X1 Slim. EFI is on beta.
- The OpenCore version - 1.0.1.
- This a beta EFI so don't expcet audio. I might fix audio in this device future.
- Everything is working except AUDIO & TRACKPAD
-  But you can use bluetooth speaker or buds to get Input/Output.
- Thanks for rakesh murpani  for testing EFI and helping. Also thanks for Olarila for initial boot.

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/'>Sonoma-OC-1.0.1-Beta1_Release-Infinix-InBook-X1 Slim</a>.</em>

**Comment by SPIRITUFA** [comment]
Hi. Does not work in sleep, the system reboots when going to sleep

**Reply by SPIRITUFA** [reply]
imessage, facetime are not authorized

**Reply by devboloji** [reply]
> Hi. Does not work in sleep, the system reboots when going to sleep
@SPIRITUFA 
nope, it won't reboot. try resetting NVRAM twice.

The face time,imessages will work via USB-Tethering / Ethernet/ heliport. if you want imessages or facetime then replace AirPortItlwm kext to itlwm(and install heliport).

**Reply by SPIRITUFA** [reply]
> nope, it won't reboot. try resetting NVRAM twice.

did not help, it also reboots when leaving in a dream

The dream started with the argument **-lilubetaall**, and the patch **changed Method(GPRW,2,N) to XPRW, paired with SSDT-GPRW.aml**
